content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -34,170 +34,43 @@ To learn about the Kubernetes networking model, see [here](/docs/concepts/servic
 
 ## How to implement the Kubernetes networking model
 
```

Comment:
So...this isn't quite right.

The network model is implemented by the container runtime(s) on each node.  The most commonly used container runtimes use CNI plugins, but that's a decision each runtime makes for itself.  In fact, some of thise links (e.g. fd.io) don't talk about CNI at all, just about the data plane.

Can you perhaps reword this paragraph?

Reply:
Also "Many different plugins exist" not "exists".

Also this is a very long sentence.  Maybe break after "vendors" and start a new sentence for "Some..."

Reply:
Thanks - I've reworded the paragraph, let me know what you think